orchestration/templates/create-json-object.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: create-json-object
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: release-date
+          - name: archive-path
+          - name: upload-path
+      {{- include "argo.retry" . | indent 6 }}
+      script:
+        image: google/cloud-sdk:slim
+        command: [bash]
+        source: |
+            json='{"release_date":"'{{ "{{inputs.parameters.release-date}}" }}'","archive_path":"'{{ "{{inputs.parameters.archive-path}}" }}'"}'
+            echo "$json" > release_history.json
+            gsutil -m rsync -r /state/release_history.json {{ "{{inputs.parameters.upload-path}}" }}
```

Previous comment got lost: I think `rsync` will complain if the source path isn't a directory. The input path is also mismatched from where the JSON is being written. This should work instead:
```suggestion
            gsutil cp release_history.json {{ "{{inputs.parameters.upload-path}}" }}
```